src/Mvc/Mvc.TagHelpers/src/InputTagHelper.cs

```diff
@@ -410,6 +431,16 @@ private TagBuilder GenerateRadio(ModelExplorer modelExplorer, IDictionary<string
             htmlAttributes);
     }
 
+    private static TagBuilder GenerateInvariantCultureMetadata(string propertyName)
```

Comment:
It would be nice to use `IHtmlGenerator` for this, but introducing new methods there would be a breaking change.

Reply:
However, we _could_ add it as an optional interface member if we're ok with adding more to the public API surface.